src/test/kotlin/de/nordgedanken/auto_hotkey/ide/documentation/AhkDocumentationProviderTest.kt

```diff
@@ -0,0 +1,103 @@
+            myFixture.editor,
+            myFixture.file,
+            element,
+            0
+        )
+        customDocumentationElement shouldNotBe null
+        customDocumentationElement shouldBe element
+    }
+
+    @ProjectDescriptor(WithOneAhkSdkAsProjDefault::class)
+    fun `test getUrlFor for command`() {
+        configureHomePath()
+        myFixture.configureByText(AhkFileType, "WinSet")
+        val element = myFixture.findElementByText("WinSet", PsiElement::class.java).firstChild
+        val url = AhkDocumentationProvider().getUrlFor(element, element)
+        url shouldBe listOf("https://www.autohotkey.com/docs/commands/WinSet.htm")
+    }
+
+    @ProjectDescriptor(WithOneAhkSdkAsProjDefault::class)
+    fun `test getUrlFor for variable`() {
+        configureHomePath()
+        myFixture.configureByText(AhkFileType, "A_LineNumber")
+        val element = myFixture.findElementByText("A_LineNumber", PsiElement::class.java).firstChild
+        val url = AhkDocumentationProvider().getUrlFor(element, element)
+        url shouldBe listOf("https://www.autohotkey.com/docs/Variables.htm#LineNumber")
+    }
+
+    @ProjectDescriptor(WithOneAhkSdkAsProjDefault::class)
+    fun `test generateDoc for command`() {
+        configureHomePath()
+        myFixture.configureByText(AhkFileType, "WinSet")
+        val element = myFixture.findElementByText("WinSet", PsiElement::class.java).firstChild
+        val doc = AhkDocumentationProvider().generateDoc(element, element)
+        doc shouldContain "<title>WinSet - Syntax &amp; Usage | AutoHotkey</title>"
+    }
+
+    @ProjectDescriptor(WithOneAhkSdkAsProjDefault::class)
+    fun `test generateDoc for variable`() {
+        configureHomePath()
+        myFixture.configureByText(AhkFileType, "A_LineNumber")
+        val element = myFixture.findElementByText("A_LineNumber", PsiElement::class.java).firstChild
+        val doc = AhkDocumentationProvider().generateDoc(element, element)
+        doc shouldContain "The number of the currently executing line within the script"
+    }
+
+    @ProjectDescriptor(WithOneAhkSdkAsProjDefault::class)
+    fun `test handleExternalLink`() {
+
+        AhkDocumentationProvider().let {
+            it.handleExternalLink(null, "#test", null) shouldBe true
+            it.handleExternalLink(null, "test", null) shouldBe true
+        }
+    }
+
+    @Test
+    fun `test canFetchDocumentationLink`() {
+        AhkDocumentationProvider().let {
+            it.canFetchDocumentationLink(null) shouldBe false
+            it.canFetchDocumentationLink("#test") shouldBe false
+            it.canFetchDocumentationLink("http") shouldBe false
+            it.canFetchDocumentationLink("test") shouldBe true
+        }
+    }
+
+    @ProjectDescriptor(WithOneAhkSdkAsProjDefault::class)
+    fun `test fetchExternalDocumentation for function`() {
+        (project.defaultAhkSdk as ProjectJdkImpl).homePath = "$testDataPath/de/nordgedanken/auto_hotkey/documentation/"
```

Comment:
It looks like line 96 above is a duplicate copy-paste of the `configureHomePath` function you already defined at the top of the class. Why not just call that function here?